lib/rules/no-duplicate-imports.js

```diff
@@ -125,18 +267,30 @@ module.exports = {
 
     create(context) {
         const includeExports = (context.options[0] || {}).includeExports,
-            importsInFile = [],
-            exportsInFile = [];
-
+            modules = {};
```

Comment:
or, alternatively, `Object.create(null)` (a "null object")

Reply:
@mdjermanovic  I used the Map object instead of the object to prevent the mentioned error and in order to perform better because our scenario (modules) involves frequent additions. @ljharb thanks for your suggestion, I think it's the easiest solution.